src/components/UploadForm/DropZone.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useMemo, useState, useEffect } from "react";
+import { useDropzone } from "react-dropzone";
+import "./style.scss";
+import {
+  baseStyle,
+  activeStyle,
+  acceptStyle,
+  rejectStyle,
+  thumbsContainer,
+  thumb,
+  thumbInner,
+  img
+} from "./stylesVarsHooks";
+export default function StyledDropzone(props) {
+  const [files, setFiles] = useState([]);
+
+  useEffect(
+    () => () => {
+      files.forEach(file => URL.revokeObjectURL(file.preview));
+    },
+    [files]
+  );
+
+  const {
+    getRootProps,
+    getInputProps,
+    isDragActive,
+    isDragAccept,
+    isDragReject,
+    acceptedFiles
+  } = useDropzone({
+    accept: "image/*",
+    onDrop: acceptedFiles => {
+      setFiles(
+        acceptedFiles.map(file =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file)
+          })
+        )
+      );
+    }
+  });
+  const filesInfo = acceptedFiles.map(file => (
+    <li key={file.path}>
+      {file.path} - {file.size} bytes
```

Comment:
does the user actually care about the byes ? or the path? we just need to show the image itself